acrobat/blocks/verb-widget/verb-widget.js

```diff
@@ -52,9 +52,9 @@ export default async function init(element) {
   const widgetHeading = createTag('h1', { class: 'verb-heading' }, children[0].textContent);
   let mobileLink = null;
   if (/iPad|iPhone|iPod/.test(window?.browser?.ua) && !window.MSStream) {
```

Comment:
Do we need an optional chaining on `window`?

Reply:
i'll fix it